proxy-example.py

```diff
@@ -0,0 +1,37 @@
+import stripe
+
+
+stripe.api_key = 'tGN0bIwXnHdwOa85VABjPdSn8nWY7G7I'
+
+print( "Attempting charge..." )
+
+stripe.proxies = {
```

Comment:
@krassif Thanks for making those changes! It looks like one last `proxies` straggler here.